server/interceptor/interceptor.go

```diff
@@ -214,11 +214,11 @@ func (tp *trillianProcessor) After(ctx context.Context, resp interface{}, handle
 		tokens = tp.info.tokens
 	} else {
 		switch resp := resp.(type) {
-		case logLeafResponse:
+		case *trillian.QueueLeafResponse:
 			if !isLeafOK(resp.GetQueuedLeaf()) {
 				tokens = 1
 			}
-		case logLeavesResponse:
+		case *trillian.QueueLeavesResponse:
 			for _, leaf := range resp.GetQueuedLeaves() {
 				if !isLeafOK(leaf) {
 					tokens++
```

Comment:
Should `AddSequencedLeavesResponse` be covered as below?

Reply:
Good spot -- that was another omission in the existing code.